nvm.sh

```diff
@@ -1044,6 +1044,20 @@ nvm_install_node_source() {
     make='gmake'
     MAKE_CXX="CXX=c++"
   fi
+
+  if [ "_$NVM_OS" = "_linux" ]; then
```

Comment:
I think we don't need to make the logical too complex, then you should also make should gmake exist ...